app/src/sharedTest/java/org/oppia/android/app/profile/PinPasswordActivityTest.kt

```diff
@@ -210,7 +210,7 @@ class PinPasswordActivityTest {
   }
 
   @Test
-  @DisableAccessibilityChecks
+  @DisableAccessibilityChecks // Disabled as this is a test file and will not be used by user
```

Comment:
@BenHenning Have a look at this comment and let us know if you think it clarifies everything.

Reply:
If if you have filed the issue then please mention that issue number as TODO.